src/content/core/okteto-variables.mdx

```diff
@@ -92,7 +92,35 @@ deploy:
 
 ### Sharing variables between commands
 
-As mentioned in the section [Scope](#scope), variables defined in a specific command item are only available to that item by default. In the following sections, there are two ways to share values between steps.
+As mentioned in the section [Scope](#scope), variables defined in a specific command item are only available to that item by default.
```

Comment:
I would use something like "each command of your deploy section", and mention destroy/test sections too

Reply:
changed to:

```diff
-... in a specific command item are only available to that item by default.
+... in each command of your deploy/destroy/test sections only available to that item by default.
```

WDYT?